src/esp/assets/ResourceManager.h

```diff
@@ -657,7 +657,7 @@ class ResourceManager {
   void joinHeirarchy(MeshData& mesh,
                      const MeshMetaData& metaData,
                      const MeshTransformNode& node,
-                     const Mn::Matrix4& transformFromParentToWorld);
+                     const Mn::Matrix4& transformFromParentToWorld) const;
```

Comment:
Could maybe make sense to specify `@param[in,out] mesh` because it's not clear that it gets modified (or maybe just `@param[out]`? not sure what exactly it does)

[Would look like this](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1MeshTools.html#af584a3b4006c42ad2aeb0f04f6127102) in the docs, for example.